html/includes/print-event-short.inc.php

```diff
@@ -7,15 +7,9 @@
 }
 
 unset($icon);
-$icon_returned = geteventicon($entry['message']);
-$icon_type = $icon_returned['icon'];
-$icon_colour = $icon_returned['colour'];
+$severity_colour = eventlog_severity($entry['severity']);
 
-if ($icon_type) {
-    $icon = "<i class='fa $icon_type fa-lg' style='color:$icon_colour' aria-hidden='true'></i>";
-} else {
-    $icon = "<i class='fa fa-bookmark-o fa-lg' style='color:black' aria-hidden='true'></i>";
-}
+$icon = "<i class='fa fa-bookmark fa-lg' style='color:$severity_colour' aria-hidden='true'></i>";
```

Comment:
OK, giving it a little thought, I dont think that would work as it would not allow for any future customization of the colours (mine might not be so great!). I think from what I can see we have 2 options,....

1. Leave it as is
2. Move the "style=color:xxx" attribute in to the function that determines the colour and just have the $severity_colour variable included?